.env

```diff
@@ -0,0 +1,3 @@
+DATABASE_URL = mongodb+srv://lawallarkson:$vMv8a3Di9tpSQY@halaqaat.sd2ol.mongodb.net/myFirstDatabase?retryWrites=true&w=majority
```

Comment:
You still commit your .env file lol

Reply:
Make sure your .gitignore file includes ``.env/``

Reply:
All sorted Chief